config/passport.js

```diff
@@ -78,28 +81,29 @@ passport.use('local-signup', new LocalStrategy({
 // by default, if there was no name, it would just be called 'local'
 
 passport.use('local-login', new LocalStrategy({
-        passReqToCallback: true // allows us to pass back the entire request to the callback
-    },
-    function (req, username, password, done) { // callback with email and password from our form
-
-        // find a user whose email is the same as the forms email
-        // we are checking to see if the user trying to login already exists
-        User.findOne({'local.username': username}, function (err, user) {
-            // if there are any errors, return the error before anything else
-            if (err)
-                return done(err);
-
-            // if no user is found, return the message
-            if (!user)
-                return done(null, false, req.flash('loginMessage', 'No user found.')); // req.flash is the way to set flashdata using connect-flash
-
-            // if the user is found but the password is wrong
-            if (!user.validPassword(password))
-                return done(null, false, req.flash('loginMessage', 'Oops! Wrong password.')); // create the loginMessage and save it to session as flashdata
-
-            // all is well, return successful user
-            return done(null, user);
-        });
+    passReqToCallback: true // allows us to pass back the entire request to the callback
+  },
+  function(req, username, password, done) { // callback with email and password from our form
+
+    // find a user whose email is the same as the forms email
+    // we are checking to see if the user trying to login already exists
+    User.findOne({
+      'local.username': username
+    }, function(err, user) {
+      // if there are any errors, return the error before anything else
+      if (err)
+        return done(err);
+
+      // if no user is found, return the message
+      if (!user)
+        return done(null, false, req.flash('loginMessage', 'No user found.')); // req.flash is the way to set flashdata using connect-flash
```

Comment:
Consider moving all your string responses to a global strings folder that can be referred to by components when necessary. This will make stuff easier to modify later on